### PR TITLE
add deck native fee rfc.

### PR DESCRIPTION
**Pull request by peerchemist**

### CODE REVIEW

File: 0006-deck-native-fee.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+There is utility in having a deck native fee, notably as card transfer fees act as deflationary force on the card supply allowing for creation of dynamic economic system revolving around token in question.
+
+## Detailed design
+
+There are three types of PeerAssets card transfer transaction:
+`card_issue` - new cards are being issued by the deck spawner,
+`card_burn` - cards are being destroyed by sending them back to deck spawn address,
+`card_transfer` - cards are being assigned to different address.
+
+Individual card is value transfer between Alice and Bob. For example: `{'sender': 'Alice', 'amount': 20, 'receiver': 'Bob'}`.
```
Comment (hrobeers) [inline 0006-deck-native-fee.md]: typo

File: 0006-deck-native-fee.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Multiple card transfers (individual cards) can be contained within the single bundle. This is result of optimization of PeerAssets protocol which allows for packing multiple card transfers in the single transaction on the native blockchain.
+Multiple cards are packed into single transaction by pairing the two lists, one is the list of the vouts and second one is list of amounts encoded in the protobuf.
+
+vouts = ['PFZwE5Ws1vhgjcNpLPobbzFNYqfXWamUmo', 'PMgLWR35CWYAj5t4LDUHww18cgHKu8yVAD', 'PPbVj2nFLsSzkY9THVQVJqF2Wn6fbKjNgt']
+amounts = [20, 14.4, 116]
+
+When two lists above are paired, it's obvious that single card on index 0 is destined to `PFZwE5Ws1vhgjcNpLPobbzFNYqfXWamUmo` and carries a amount of 20. Sender is the sender of the transaction.
+
+Deck native fee requires all card transfer bundles to start with the card_burn transaction, that is it has to be addressed to deck issuer and amount has to be equal or larger than deck's native fee.
```
Comment (hrobeers) [inline 0006-deck-native-fee.md:40]: So the fee is burned, meaning that if `ONCE` issue mode is activated, the tokens cannot be reissued by the issuing party.
This means fees are paid to all holders of the token and cannot be used to fund development (fund the issuer), unless if the `MULTI` mode is specified.
We might introduce a new issue mode that allows the re-issuance of burned coins, which is more strict than `MULTI` but less strict than `ONCE`?

File: 0006-deck-native-fee.md (hunk 1 above)
Comment (peerchemist) [reply 0006-deck-native-fee.md]: Where is the typo?

File: 0006-deck-native-fee.md (hunk 2 above)
Comment (peerchemist) [reply 0006-deck-native-fee.md:40]: Yes that is correct, there is no token re-issuance possible. The issuer is rewarded by diminishing supply and increasing scarcity of tokens just as anyone else.
Re-issuance of fees (burned tokens) sounds interesting.

File: 0006-deck-native-fee.md (hunk 1 above)
Comment (hrobeers) [reply 0006-deck-native-fee.md]: Something seems wrong in this sentence:
> Individual card is value transfer between ...

Shouldn't this be something like:
> Individual card **transfer is a** value transfer between ...

Comment (peerchemist) [reply 0006-deck-native-fee.md]: +1

**Pull request merged**